smarts/core/smarts.py

```diff
@@ -1053,9 +1053,15 @@ def _try_emit_envision_state(self, provider_state, obs, scores):
             for bubble in self._bubble_manager.bubbles
         ]
 
+        scenario_name = {
+            agent_id: os.path.split(self.scenario._root)[1]
+            for agent_id in list(self._agent_manager.ego_agent_ids)
+        }
+
```

Comment:
I wonder if this could somehow be just a string since the scenario is the same for all agents. I wouldn't know how to handle this on the JS side though. Other than that it looks good!

Reply:
The file changes do address this issue. If you run the simulation with the new web dist/ then you will see only the scenario name and not an agent_id tag beside it